src/server/services/paddle.service.ts

```diff
@@ -184,23 +185,43 @@ export const processCompleteBuzzTransaction = async (
   const userId = meta.user_id ?? meta.userId;
   const { purchasesMultiplier } = await getMultipliersForUser(userId);
   const amount = meta.buzz_amount ?? meta.buzzAmount;
-  const buzzAmount = Math.ceil(amount * (purchasesMultiplier ?? 1));
+
+  const { blueBuzzAdded, totalYellowBuzz } = getBuzzBulkMultiplier({
+    buzzAmount: amount,
+    purchasesMultiplier,
+  });
 
   // Pay the user:
-  const buzzTransaction = await createBuzzTransaction({
-    amount: buzzAmount,
+  await createBuzzTransaction({
+    amount: totalYellowBuzz,
     fromAccountId: 0,
     toAccountId: userId,
     externalTransactionId: transaction.id,
     type: TransactionType.Purchase,
     description: `Purchase of ${amount} Buzz. ${
       purchasesMultiplier && purchasesMultiplier > 1 ? 'Multiplier applied due to membership. ' : ''
-    }A total of ${buzzAmount} Buzz was added to your account.`,
+    }A total of ${totalYellowBuzz} Buzz was added to your account.`,
```

Comment:
Probably good idea to add commas
```suggestion
    }A total of ${numberWithCommas(totalYellowBuzz)} Buzz was added to your account.`,
```